BUILDING.md

```diff
@@ -92,6 +92,37 @@ cargo make -e BUILDSYS_ARCH=my-arch-here
 
 (You can use variant and arch arguments together, too.)
 
+#### Packeges licenses
+
+Most of the packages included in the base OS image are distributed using the licenses included in the upstream sources.
+However, there are some cases where multiple licenses could apply to a package, depending on who will ditribute the OS images.
+Bottlerocket's build system uses the `Licenses.toml` file in conjuction with the `licenses` directory to configure the licenses used for such special packages.
+You can use them to specify what licenses better suits your needs when you build and distribute your own Bottlerocket images.
+Here is an example of a simple `Licenses.toml` configuration file:
+
+```toml
+[package]
+spdx-id = "SPDX-ID"
+licenses = [
+  { path = "the-license.txt" }
+]
+```
+
+In the previous example, it is expected that the file `the-license.txt` is present in `licenses`.
+You can retrieve the licenses from a remote endpoint, or the local filesystem if you specify the `license-url` field:
+
+```toml
+spdx-id = "SPDX-ID AND SPDX-ID-2" # Package with multiple licenses
```

Comment:
```suggestion
```toml
[package]
spdx-id = "SPDX-ID AND SPDX-ID-2" # Package with multiple licenses
```